salt/modules/boto_cloudfront.py

```diff
@@ -263,8 +263,8 @@ def export_distributions(region=None, key=None, keyid=None, profile=None):
         # as opposed to being called from execution or state modules
         log.trace("Boto client error: {}", exc)
 
-    dumper = __utils__["yaml.get_dumper"]("IndentedSafeOrderedDumper")
-    return __utils__["yaml.dump"](
+    dumper = __utils__["yamldumper.get_dumper"]("IndentedSafeOrderedDumper")
```

Comment:
**question** do all these need to be changed?
```

✦2 ❯ ag '"yaml\.' salt
salt/modules/boto_iam.py
1682:    return __utils__["yaml.safe_dump"](results, default_flow_style=False, indent=2)
1728:    return __utils__["yaml.safe_dump"](results, default_flow_style=False, indent=2)
```

I'm assuming that this only applies for `__utils__` and not `__serializers__` - I just went and found a couple of those, specifically in salt/states/file.py - it looks for serialzer_name in __serializers__ where `yaml.serialize` is what's called, which doesn't look like the yamldumper :thinking:

Reply:
The `yamldumper` only applies to the utils dunder.
They were never using our `yamldumper` utils module, they were using the actual `yaml` package.